ansible/roles/airflow/files/docker-trampoline

```diff
@@ -92,12 +93,20 @@ case "$script" in
     set -- /usr/local/bin/cleanup_reports_raw.py --bucket "${bucket}" --reports-raw-root "$private/reports-raw" --canned-root "$private/canned" --s3-ls "$private/reports-raw-s3-ls/${bucket}.json.gz"
     ;;
 
+    sanitised_s3_cp)
+    docknet="" # no `--network none'
+    volargs="${volargs} --env-file=/etc/af-worker/s3read.env"
+    volargs="${volargs} --volume=$public/sanitised/${bucket}:$public/sanitised/${bucket}:ro"
+    script=""
+    set -- /usr/local/bin/aws_s3_cp.py --src "$public/sanitised/${bucket}" --dst "s3://ooni-data/sanitised/${bucket}"
+    ;;
+
```

Comment:
You're either kidding me or have not looked at the data and/or misunderstand how it works (every option is OK, but I truly don't understand what one of them is the case).
It obviously does not work like that, the `.../sanitized/2017-10-*` directories are empty.

Reply:
1. Chillax
2. Most directories on datacollector are in fact empty, but I was assuming this was due to some bug in the new pipeline as I do see some of them containing files. Example:
```
$ ls  -1 /data/ooni/public/sanitised/2017-02-24 | wc -l
2470
```
3. If that is not the correct place to get these from, please update the path to what is the correct path to get them from.